src/layouts/Upgrade.tsx

```diff
@@ -53,10 +52,6 @@ const Title = (props: ChildOnlyProp) => (
   />
 )
 
-const SummaryPoint = (props: ChildOnlyProp) => (
-  <ListItem color="text300" mb={0} {...props} />
```

Comment:
I think, at very least, we want to keep the margin-bottom here. I'd also advocate for not using pure white here. 

Before:

<img width="740" alt="Screenshot 2024-04-12 at 16 39 00" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/06b5b19d-97c6-42d2-8599-650ef7c31a1b">


After:

<img width="740" alt="Screenshot 2024-04-12 at 16 38 46" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/777033f4-0da0-4ee8-ad45-64ab340a7e8d">


Maybe we can allow these changes to be handled in this discussion: https://github.com/ethereum/ethereum-org-website/pull/12639#issuecomment-2045619701

cc: @nloureiro